src/main/java/mate/academy/bookstore/repository/user/UserRepository.java

```diff
@@ -2,12 +2,15 @@
 
 import java.util.Optional;
 import mate.academy.bookstore.model.User;
+import org.springframework.data.jpa.repository.EntityGraph;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface UserRepository extends JpaRepository<User, Long> {
-    Optional<User> findByEmail(String email);
+    @EntityGraph(attributePaths = "roles")
+    Optional<User> findByEmail(@Param("email") String email);
```

Comment:
```suggestion
    Optional<User> findByEmail(String email);
```
redundant annotation